scripts/flitcli/flit_bisect.py

```diff
@@ -447,111 +530,239 @@ def memoized_func(strlist):
         return value
     return memoized_func
 
-def bisect_search(is_bad, elements, found_callback=None):
+def bisect_biggest(score_func, elements, found_callback=None, k=1):
+    '''
+    Performs the bisect search, attempting to find the biggest offenders.  This
+    is different from bisect_search() in that this function only tries to
+    identify the top k offenders, not all of them.  If k is less than or equal
```

Comment:
I thought the biggest search was more expensive if the argument k is greater than or equal to the number of offenders.